bin/polly

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+/usr/bin/env python3 `dirname $0`/polly.py "$@"
```

Comment:
Will not work if `polly` called through symbolic link. Should be like

```
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

See [SO: Can a Bash script tell what directory it's stored in?](http://stackoverflow.com/a/246128/2288008)